eZ/Publish/API/Repository/Tests/FieldType/CheckboxIntegrationTest.php

```diff
@@ -392,6 +387,21 @@ public function testFindContentFieldCriterion(bool $isActive): void
         self::assertSame($isActive, $value->bool);
     }
 
+    public function testAddFieldDefinition()
+    {
+        $content = $this->addFieldDefinition();
+
+        $this->assertCount(2, $content->getFields());
```

Comment:
Would like to see sanity check prior `addFieldDefinition` - assert count = 1

Reply:
Not sure how to do this - this test is a variation of the base one:
https://github.com/ezsystems/ezplatform-kernel/blob/59ca1434ce8fb5169d39db258dfdc2ebc249120b/eZ/Publish/API/Repository/Tests/FieldType/BaseIntegrationTest.php#L1009-L1022

But I had to change the assertion.

The method 'addFieldDefinition` first removes the fieldDefinition:
https://github.com/ezsystems/ezplatform-kernel/blob/59ca1434ce8fb5169d39db258dfdc2ebc249120b/eZ/Publish/API/Repository/Tests/FieldType/BaseIntegrationTest.php#L985
and then adds it again

The flow inside `addFieldDefinition` looks like this then:
- The Content item is created (with 2 fields)
- the tested field definition is removed
- the tested field definition is added again

Current approach makes the `$content` variable available only after the call to `addFieldDefinition` finishes, I can't really add that sanity check without reworking this test.

I wanted to reuse the existing setup methods, but if you think it's good to add some custom setup for this test then I will do it